lib/experimental/webfonts-utils.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * Stores utility functions used by the Webfonts API.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Transforms the source of the font face from `file.:/` into an actual URI.
+ *
+ * @since 6.0.0
+ *
+ * @param array $font_face The font face.
+ * @return array The URI-resolved font face.
+ */
+function _gutenberg_resolve_font_face_uri( array $font_face ) {
```

Comment:
Are all of these functions that start with an underscore `_` intended to be private? I assume yes but wanted to check before adding the private annotation.

Reply:
Yes, they are meant to be private since they are only used internally by the API.

Reply:
`@private` annotation added in https://github.com/WordPress/gutenberg/pull/39988/commits/d6bbb43c1a3b09a7227c940ac90310a806baedd2